pkg/provider/data_source_region.go

```diff
@@ -90,6 +90,23 @@ func (r *regionsDataSource) Schema(ctx context.Context, req datasource.SchemaReq
 				Description: "Unique region ID. For example, \"germanywestcentral\" in the Azure cloud provider, \"eu-west-1\" in the AWS cloud provider.",
 				Optional:    true,
 			},
+			"tags": schema.SetNestedAttribute{
+				Description: "Assign existing tags or create tags to assign to this resource",
```

Comment:
as this is a data_source description we should not mention assign or create, instead it should mention something like "show tags associated with this resource"

Reply:
I will change it all in a later PR https://github.com/EnterpriseDB/terraform-provider-biganimal/pull/583 so it is done altogether